htdocs/public/ticket/ajax/ajax.php

```diff
@@ -65,6 +66,12 @@
 	httponly_accessforbidden('Option TICKET_CREATE_THIRD_PARTY_WITH_CONTACT_IF_NOT_EXIST of module ticket is not enabled');
 }
 
+if (empty($token) || $token !== currentToken()) {
```

Comment:
Because the page that is calling the ajax is a public page, using a token is useless because the token will be also public.
Goal of token is to avoid to allow CSRF attack so allow an attacker to execute the page into the session of another user. But here this is a public component and any body can execute it so what is goal of blocking the use into the session of another user when you are already allowed to execute it in your own session.